src/Compute/Compute/help/New-AzHostGroup.md

```diff
@@ -161,6 +161,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -SupportAutomaticPlacement
```

Comment:
what is the default status of AutomaticPlacement without use specifying?

Reply:
Thanks, added the default value